packages/usa-nav/src/test/navigation.spec.js

```diff
@@ -163,9 +180,10 @@ describe("navigation toggle", () => {
 
   it("collapses dropdowns when focus leaves nav", () => {
     menuButton.click();
-    navLink.click();
-    EVENTS.focusOut(navLink);
-    assert.strictEqual(isVisible(nav), false);
```

Comment:
## Invalid test

This test would not properly test dropdowns closing on focus out for a few reasons

### 1. `navLink.click()` closes the nav

Clicking the navigation link closes the entire navigation, not just the dropdowns (consider mobile views). 

To illustrate this, removing `EVENTS.focusOut(navLink)`still causes test to pass.

### 2. Incorrect testing instructions

Instead of testing that the entire nav is closed, we should be checking if the navigation dropdown/accordion is closed.

I was able to emulate this by instead focusing out of `navPrimary` (emulating focusing the search bar for example) and checking the `aria-expanded` attribute.